docs/samples/kafka/README.md

```diff
@@ -157,6 +126,49 @@ kafkasource-kafka-source-3d809fe2-1267-11ea-99d0-42010af00zbn5h   1/1     Runnin
 
 ## Upload a digit image to Minio mnist bucket
 The last step is to upload the image `images/0.png`, image then should be moved to the classified bucket based on the prediction response!
+```bash
+mc cp images/0.png myminio/mnist
+```
+you should expect a notification event like following sent to kafka topic `mnist` after uploading an image in `mnist` bucket
+```json
+{
+   "EventType":"s3:ObjectCreated:Put",
+   "Key":"mnist/0.png",
+   "Records":[
+      {"eventVersion":"2.0",
+       "eventSource":"minio:s3",
+       "awsRegion":"",
+       "eventTime":"2019-11-17T19:08:08Z",
+       "eventName":"s3:ObjectCreated:Put",
+       "userIdentity":{"principalId":"minio"},
+       "requestParameters":{"sourceIPAddress":"127.0.0.1:37830"},
+       "responseElements":{"x-amz-request-id":"15D808BF706E0994",
+       "x-minio-origin-endpoint":"http://10.244.0.71:9000"},
+       "s3":{
+          "s3SchemaVersion":"1.0",
+          "configurationId":"Config",
+          "bucket":{
+               "name":"mnist",
+               "ownerIdentity":{"principalId":"minio"},
+               "arn":"arn:aws:s3:::mnist"},
+          "object":{"key":"0.png","size":324,"eTag":"ebed21f6f77b0a64673a3c96b0c623ba","contentType":"image/png","userMetadata":{"content-type":"image/png"},"versionId":"1","sequencer":"15D808BF706E0994"}},
+          "source":{"host":"","port":"","userAgent":""}}
+   ],
+   "level":"info",
+   "msg":"",
+   "time":"2019-11-17T19:08:08Z"
+}
+```
 
-
+Check the transformer log, you should expect a prediction response and put the image to the corresponding bucket
+```bash
+kubectl logs mnist-transformer-default-rctjm-deployment-54d59c849c-2dq98  kfserving-container
+[I 201128 22:32:27 kfserver:88] Registering model: mnist
+[I 201128 22:32:27 kfserver:77] Listening on port 8080
+[I 201128 22:32:27 kfserver:79] Will fork 0 workers
+[I 201128 22:32:27 process:123] Starting 6 processes
+[I 201128 22:32:44 connectionpool:203] Starting new HTTP connection (1): minio-service
+[I 201128 22:32:58 image_transformer:51] {'predictions': [{'predictions': [0.0247901566, 1.37231364e-05, 0.0202635303, 0.39037028, 0.000513458275, 0.435112566, 0.000607515569, 0.00041125578, 0.127784252, 0.000133168287], 'classes': 5}]}
+[I 201128 22:32:58 image_transformer:53] digit:5
+```
```

Comment:
In testing this failed as bucket folders were not created to hold digits. You need to add to docs the need to create bucket folders, e.g. `mc mb myminio/digit-0` -> `mc mb myminio/digit-9`

Reply:
add the setup when setting up minio 
`mc mb myminio/digit-[0-9]`